service/controller/app/app.go

```diff
@@ -112,7 +101,7 @@ func NewApp(config Config) (*App, error) {
 			ResourceSets: []*controller.ResourceSet{
 				resourceSetV1,
 			},
-			RESTClient: config.G8sClient.CoreV1alpha1().RESTClient(),
+			RESTClient: config.K8sClient.G8sClient().CoreV1alpha1().RESTClient(),
```

Comment:
The e2e tests broke when I used `config.K8sClient.RESTClient()` so I suspect something is wrong with the scheme builder.

I've reverted as I don't have time to investigate further. I'm off tomorrow so I need to get my PRs merged.

Reply:
No worries. We dropped the rest client there anyway. Will be gone in the next step.